docs/src/main/paradox/scala/http/routing-dsl/index.md

```diff
@@ -36,6 +37,18 @@ This is a complete, very basic Akka HTTP application relying on the Routing DSL:
 
 It starts an HTTP Server on localhost and replies to GET requests to `/hello` with a simple response.
 
+@@@ warning { title="API may change" }
+The following code example uses an experimental feature and its API is subjected to change in futures releases of Akka HTTP.
+For further information about this marker, see [The @DoNotInherit and @ApiMayChange markers](http://doc.akka.io/docs/akka/current/common/binary-compatibility-rules.html#The_@DoNotInherit_and_@ApiMayChange_markers)
+in the Akka documentation.
+@@@
+
+In order to help getting started with setting up a server Akka HTTP comes with an experimental helper class called `HttpApp`.
```

Comment:
Nitpicking: `To help start a server Akka HTTP provides an experimental helper class called `HttpApp`.